test/cli/test-other.py

```diff
@@ -505,6 +508,32 @@ class _clz {
     expect.sort()
     assert lines == expect
 
+    # Perform same analysis again, but now in standalone mode.
```

Comment:
We should probably move all the standalone addon tests from being a non-portable, inline command-line script to Python.

I also think we should move the addon tests to a separate script. That is probably true for several things in `test-other.py` which is basically just a dumping ground for most non-project tests.

Both is nothing for this PR but for future cleanups.

Reply:
> Both is nothing for this PR but for future cleanups.

Fine with me to do this while I'm at it...

Reply:
I force pushed a split of `test-other.py` into `test-addons.py` on this PR.